.github/workflows/resource-indexer.yml

```diff
@@ -0,0 +1,39 @@
+name: rebuild the resource collection index
```

Comment:
Minor, but I'd name this something slightly less unwieldy in the GitHub Actions web UI.  Maybe just "resource indexer" or "index resources"?

Reply:
Sure - done